.release-version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Thanks, you're right.  This will be version 1.0.0 though.  Not an alpha.  This will be used on production data from day 1.  The existing numbering strategy has been version 1.0.0 from day 1 on PSD applications.